tests/kernel/sched/schedule_api tests fail to build

When we do:
`./scripts/sanitycheck -p qemu_cortex_m3 -T tests/kernel/sched/schedule_api/`

We get:

```
4 tests selected, 804 tests discarded due to filters
total complete:    0/   4   0%  failed:    0

qemu_cortex_m3            tests/kernel/sched/schedule_api/kernel.sched       FAILED: build_error
	see: sanity-out/qemu_cortex_m3/tests/kernel/sched/schedule_api/kernel.sched/build.log

total complete:    1/   4  25%  failed:    1

qemu_cortex_m3            tests/kernel/sched/schedule_api/kernel.sched_no_timeslicing FAILED: build_error
	see: sanity-out/qemu_cortex_m3/tests/kernel/sched/schedule_api/kernel.sched_no_timeslicing/build.log
```

```
/home/galak/git/zephyr/kernel/sched.c:759:3: error: #error Too many priorities for multiqueue scheduler (max 32)
 # error Too many priorities for multiqueue scheduler (max 32)
   ^~~~~
```

Not sure if we need to have 'CONFIG_SCHED_SCALABLE=y' enabled in `tests/kernel/sched/schedule_api/prj.conf` or something else.

--- a/tests/kernel/sched/schedule_api/testcase.yaml
+++ b/tests/kernel/sched/schedule_api/testcase.yaml
@@ -1,10 +1,12 @@
 tests:
   kernel.sched:
+    filter: not CONFIG_SCHED_MULTIQ
     extra_configs:
       - CONFIG_TIMESLICING=y
     min_ram: 40
     tags: kernel threads sched userspace
   kernel.sched_no_timeslicing:
+    filter: not CONFIG_SCHED_MULTIQ
     extra_configs:
       - CONFIG_TIMESLICING=n
     min_ram: 40

